src/gui/SettingsWidgetGeneral.ui

```diff
@@ -56,54 +56,80 @@
     </widget>
    </item>
    <item row="5" column="0">
+    <widget class="QLabel" name="label_2">
```

Comment:
The name of this widget is generic (`label_2`) should be something else with sense.